frontend/src/app/AppRouter.tsx

```diff
@@ -6,6 +6,7 @@ import {
   ADMINFORM_ROUTE,
   ADMINFORM_SETTINGS_SUBROUTE,
   LOGIN_ROUTE,
+  PRIVACY_ROUTE,
```

Comment:
nit: can probably rename to `PRIVACY_POLICY_ROUTE` to make it clear what this is for haha

Reply:
62968cb